src/include/86box/sound.h

```diff
@@ -71,6 +77,8 @@ extern void closeal(void);
 extern void inital(void);
 extern void givealbuffer(void *buf);
 extern void givealbuffer_cd(void *buf);
+extern void givealbuffer_source(void *buf, al_source* src, int size, int freq);
+extern al_source* al_create_sound_source(uint32_t freq, uint32_t buffer_size_mono);
```

Comment:
Given that you are already messing with these parts, would it be possible to change the function names to something a bit more generic?

Reply:
al makes more sense since it's the audio library we are interfacing with.

Reply:
not if XAudio2/FAudio is enabled...

Reply:
They count as audio libraries as well.

Reply:
Except that `al` in these functions' names refers to Open**AL** specifically, not just any audio library.

Reply:
Well, it turns out this PR has more problems: My Windows 98 SE VM drops from 100% to 92% on the POST screen when the NCR 53c8xx SCSI BIOS runs, with this PR, so my intuition that this was going to slightly slowdown the emulator overall, has been correct.